en/flight_controller/pixhawk4.md

```diff
@@ -58,6 +58,10 @@ Order from [Holybro](https://shop.holybro.com/pixhawk-4beta-launch_p1089.html).
 
 Download *Pixhawk 4* pinouts from [here](http://www.holybro.com/manual/Pixhawk4-Pinouts.pdf).
 
+> **Note** Connector pin assignments are left to right (i.e. Pin 1 is the left-most pin).
+  The exception is the [debug port(s)](#debug_port) (pin 1 is the right-most, as shown below).
+  
```

Comment:
@hamishwillee this is relative to the connector mounting and orientation. Is there a picture?

Reply:
@davids5 I am assuming that for all-but-debug is relative to a connector with this orientation 
![image](https://user-images.githubusercontent.com/5368500/76266721-81d7d200-62bc-11ea-8ca7-2f7cbfc83d06.png)
- is the statement correct for this orientation
- Is there a "standard" way to refer to this orientation?


For the debug port, there is an image in the debug section (below). Again, I'm not sure if there is a good way to refer to the orientation.

![image](https://user-images.githubusercontent.com/5368500/76266811-d24f2f80-62bc-11ea-92de-9dfe3ba99564.png)